### PR TITLE
[ko] Korean Translation for 3-transport lesson 4

### PR DESCRIPTION
**Pull request by WZNT-KimHyeJin**
Add README.ko.md

### CODE REVIEW

**Comment by WZNT-KimHyeJin** [comment]
@jhYun505 I need your feedback 🍀

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+이 비디오는 Geofence에 대한 개요와 Azure Maps에서 Geofence를 사용하는 방법을 제공하며, 이 레슨에서 다룰 주제는 아래와 같습니다.
+
+[![Geofencing with Azure Maps from the Microsoft Developer IoT show](https://img.youtube.com/vi/nsrgYhaYNVY/0.jpg)](https://www.youtube.com/watch?v=nsrgYhaYNVY)
+
+> 🎥 영상을 시청하려면 이미지를 클릭하세요
+
+## 강의 전 퀴즈
+
+[강의 전 퀴즈](https://black-meadow-040d15503.1.azurestaticapps.net/quiz/27)
+
```
Comment (DobiIsFree) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:16]: '개요' 로 통일했어요!

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+    이전 작업의 지오펜스의 UDID로 `<UDID>`를 바꿉니다.
+
+    `<lat>`와 `<lon>`을 테스트할 위도와 경도로 바꿉니다.
+    
+   이 URL은 `https://atlas.microsoft.com/spatial/geofence/json`API를 통해 GeoJSON을 사용하여 정의된 Geofence를 쿼리합니다. `1.0` 버전 이상의 API 버전을 대상으로 하며, `deviceId`의 매개변수는 필수이고, 위도와 경도가 나오는 장치의 이름이어야 합니다.
+
+   기본 검색 버퍼는 50m이며, `searchBuffer=<distance>`와 같은 추가 매개변수를 전달하여 이 값을 변경할 수 있습니다. `<distance>` 는 검색 버퍼 거리(0~500 사이값, m 단위)로 설정합니다.
+
+1. curl을 사용하여 다음 URL에 GET 요청을 작성합니다.
```
Comment (DobiIsFree) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:224]: 숫자가 바뀌어야 할 것 같아요

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+
+[![Geofencing with Azure Maps from the Microsoft Developer IoT show](https://img.youtube.com/vi/nsrgYhaYNVY/0.jpg)](https://www.youtube.com/watch?v=nsrgYhaYNVY)
+
+> 🎥 영상을 시청하려면 이미지를 클릭하세요
+
+## 강의 전 퀴즈
+
+[강의 전 퀴즈](https://black-meadow-040d15503.1.azurestaticapps.net/quiz/27)
+
+## 도입
```
Comment (DobiIsFree) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:17]: '개요' 로 통일해주세요 :)

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+   geofence  gps-sensor
+   ```
+
+> 💁 이전 강의에서 IoT Hub 이벤트 모니터를 실행하면 `$Default` 소비자 그룹에 연결되었습니다. 이 때문에 이벤트 모니터와 이벤트 트리거를 실행할 수 없었습니다. 두 가지를 모두 실행하려면 모든 기능 앱에 대해 개별적인 소비자 그룹을 사용하고, 이벤트 모니터에 대해 `$Default`를 유지해야합니다.
+
+### 작업 - 새 IoT 허브 트리거 생성
+
+1. 이전 강의에서 만든 `gps-trigger` 기능 앱에 새로운 IoT 허브 이벤트 트리거를 추가합니다. 이 함수를 지오펜스 트리거라고 합니다.
+
+   > ⚠️ 필요한 경우 [project 2, lesson 5 IoT 허브 이벤트 트리거를 생성하기 위한 지침](../../../2-farm/lessons/5-migrate-application-to-the-cloud/README.md#create-an-iot-hub-event-trigger)에서 참고할 수 있습니다..
```
Comment (jhYun505) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:321]: - two dots(..)
- hyperlink not work(maybe ../../../../2-farm will be work!)

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+
+1. `for events in events` 루프 안에 다음을 추가하여 각 이벤트의 위도와 경도를 얻습니다.
+
+   ```python
+   event_body = json.loads(event.get_body().decode('utf-8'))
+   lat = event_body['gps']['lat']
+   lon = event_body['gps']['lon']
+   ```
+
+  이 코드는 JSON을 이벤트 본문에서 사전으로 변환한 다음 `gps` 필드에서 `lat`와 `lon`을 추출한다.
```
Comment (jhYun505) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:386]: 한다 -> 합니다.

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+       logging.info(f'Point is just inside geofence by a distance of {distance}m')
+   ```
+
+   이 코드는 1개의 지오메트리를 가정하고 해당 단일 지오메트리에서 거리를 추출합니다. 그런 다음 거리를 기준으로 다양한 메시지를 기록합니다.
+
+1. 이 코드를 실행하십시오. GPS 좌표가 지오펜스 범위 안에 있는지, 점이 50m 이내에 있는지에 대한 여부가 로깅 출력에 표시됩니다. GPS 센서의 위치를 기반으로 다른 지오펜스를 사용하여 이 코드를 시도하고, 휴대폰에서 WiFi에 연결되거나 가상 IoT 장치에서 다른 좌표를 사용하여 센서를 움직여 보세요.
+
+1. 준비가 되었으면 이 코드를 클라우드의 Functions 앱에 배포하십시오. 새 응용 프로그램 설정을 배포하는 것을 잊지 마십시오.
+
+   > ⚠️ 필요한 경우 [project 2, lesson 5 의 응용 프로그램 설정 업로드 지침](../../../2-farm/lessons/5-migrate-application-to-the-cloud/README.md#task---upload-your-application-settings)을 참고하십시오.
```
Comment (jhYun505) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:435]: hyperlink not work

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 7]
```diff
@@ -0,0 +1,461 @@
...
+
+   > ⚠️ 필요한 경우 [project 2, lesson 5 Functions 앱을 배포하기 위한 지침](../../../2-farm/lessons/5-migrate-application-to-the-cloud/README.md#task---deploy-your-functions-app-to-the-cloud)을 참고하십시오.
```
Comment (jhYun505) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:437]: here too

File: 3-transport/lessons/4-geofences/translations/README.ko.md [hunk 8]
```diff
@@ -0,0 +1,461 @@
...
+
+> 💁 [code/functions](code/functions)폴더에서 코드를 확인할 수 있습니다.
```
Comment (jhYun505) [inline 3-transport/lessons/4-geofences/translations/README.ko.md:439]: this should be (../code/functions)

**Review by jhYun505 — CHANGES_REQUESTED** [review]
Perfect except for a few minor mistakes!

**Comment by WZNT-KimHyeJin** [comment]
Thank you for spending your time!

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett I finished the translation! Please check this file before you merge! Thank you 🍀

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett I think I made several Pull Requests so there's a conflict. Thank you for letting me know.
And if you have the same conflict when you merge the #434  PR,  please mention me anytime you are free! 
Thank you!

**Review by jimbobbennett — APPROVED** [review]

**Comment by WZNT-KimHyeJin** [comment]
@jimbobbennett Yes! You can merge this!!

**Comment by jimbobbennett** [comment]
Merged! Appreciating all this amazing work from y'all.

**Pull request merged**